packages/ui/src/components/Dialog/LargeDialog.css.ts

```diff
@@ -0,0 +1,40 @@
+import { style } from '@vanilla-extract/css'
+import { minWidth, tokens } from '../../theme'
+
+export const closeButton = style({
+  position: 'absolute',
+  right: tokens.space.md,
+  top: tokens.space.md,
+})
+
+export const content = style({
+  display: 'flex',
+  width: '100%',
+})
+
+export const card = style({
+  position: 'relative',
+  marginBlock: 'auto',
+  marginInline: tokens.space.md,
```

Comment:
it seem to have 24px padding in [Figma](https://www.figma.com/design/p1TEzLLVqJB9gKDNTDVGU4/Hedvig.com?node-id=3400-127452&m=dev)
```suggestion
  marginInline: tokens.space.lg,
```

Reply:
This represents outer margin. I'll make sure inner padding is 1.5rem